erica/config.py

```diff
@@ -28,8 +28,10 @@ class Settings(BaseSettings):
     hersteller_id: str = Field("74931", env='ELSTER_HERSTELLER_ID')
     queue_host: str = Field("localhost", env='QUEUE_HOST')
     queue_port: int = Field(6379, env='QUEUE_PORT')
-    default_queues: List[str] = ['dongle', 'cert', 'common']
+    default_queues: str = 'dongle, cert, common'
```

Comment:
List import  not longer needed

Reply:
List was replaced by a simple string with the values separated with comma, is this way ok?

Reply:
I see no problems, but the former implementation with list comes from @JulianRoesner  I think.

Reply:
Yes, the change is fine. The former was not to be honest.

I am going to remove the import.